cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class ShippingStepPage {
+    private CheckTerm: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.CheckTerm = '#cgv';
+        this.CartNavigation = '.cart_navigation span'
+    
+
+    }
+
+    public PressTerm(): void {
+        cy.get(this.CheckTerm).click()
+    }
+
+    public PressNavigation(): void {
```

Comment:
This is referring to click on the proceed to checkout button, let's name the method in a way that indicate that we are proceeding to checkout